src/hooks/useApi.ts

```diff
@@ -51,13 +51,7 @@ const API: {
     f: Commodity.find,
   },
   '/api/accounts': {
-    f: queries.getAccountsWithPath,
-  },
-  '/api/accounts/splits': {
-    f: () => queries.getAccountsWithPath({
-      relations: { splits: { fk_transaction: true } },
-      showRoot: true,
-    }),
+    f: q.getAccounts,
```

Comment:
yes, I'm deleting the extra `/api/accounts/splits`. This is related to the work in #132 as I'm trying to simplify the amount of operations/api entries we have. The use case is that we load the accounts at start both for showing data and the selector. Before this caused two queries that return mostly the same data. We just do one now.